src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -87,63 +87,63 @@
  * @since 01/03/17
  */
 @Component
-public class AzureProvider implements CloudProvider {
+public abstract class AzureProvider implements CloudProvider {
```

Comment:
I don t see the implementation of this abstract class

Reply:
AzureVMsProvider.java and AzureScaleSetProvider.java you have to click on "load diff" on github to see them.